src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -491,6 +494,24 @@ def setup_default_btrfs_subvolumes
         add_btrfs_subvolumes(spec.subvolumes) if spec.subvolumes
       end
 
+      # Display name to represent the filesystem
+      #
+      # Only multidevice Btrfs has its own representation
+      #
+      # @return [String, nil]
+      def display_name
+        return nil unless multidevice?
+
+        textdomain "storage"
+
+        # TRANSLATORS: display name when the Btrfs is multidevice
+        format(
+          _("Btrfs %{name} over %{num_devices} devices"),
```

Comment:
The "+" sign makes this sentence hard to read

```
Btrfs sda1+ over 2 devices
```

I would prefer an alternative which doesn't imply that plus sign being in the middle. For example:

```
Btrfs over 2 devices (sda1+)
```

Reply:
And something like:

```
Btrfs over 2 devices (sda1 ...)
```

Reply:
ok